Prototope/Color.swift

```diff
@@ -50,23 +57,23 @@ public struct Color {
 	}
 
 	/** Constructs a Color from a UIColor. */
-	init(_ uiColor: UIColor) {
+	init(_ uiColor: SystemColor) {
```

Comment:
Probably worth changing this argument and property name to, like, `systemColor`.